src/en/readcomic/src/eu/kanade/tachiyomi/extension/en/readcomic/Readcomic.kt

```diff
@@ -0,0 +1,201 @@
+
+    override val supportsLatest = true
+
+    override fun popularMangaSelector() = ".manga-box"
+
+    override fun latestUpdatesSelector() = ".home-list .hl-box .hlb-name"
+
+    override fun popularMangaRequest(page: Int): Request {
+        return GET("$baseUrl/popular-comic/$page")
+    }
+
+    override fun latestUpdatesRequest(page: Int): Request {
+        return GET("$baseUrl/comic-updates/$page")
+    }
+
+    override fun popularMangaFromElement(element: Element): SManga =
+        SManga.create().apply {
+            setUrlWithoutDomain(element.selectFirst("a")!!.attr("abs:href"))
+            title = element.selectFirst("h3")!!.text()
+            thumbnail_url = element.selectFirst(".image")!!.selectFirst("img")!!.attr("abs:src")
+        }
+
+    override fun latestUpdatesFromElement(element: Element): SManga =
+        SManga.create().apply {
+            setUrlWithoutDomain(element.attr("abs:href"))
+            title = element.text()
+        }
+
+    override fun popularMangaNextPageSelector() = ".general-nav a:last-child"
+
+    override fun latestUpdatesNextPageSelector() = popularMangaNextPageSelector()
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request { // publisher > writer > artist + sorting for both if else
+        var statusFilterText = ""
+        var included = ""
+        var excluded = ""
+
+        if (!filters.isEmpty()) {
+            for (filter in filters) {
+                when (filter) {
+                    is StatusFilter -> {
+                        if (filter.state == 1) {
+                            statusFilterText = "CMP"
+                        } else if (filter.state == 2) statusFilterText = "ONG"
+                    }
+                    is GenreList -> {
+                        included = filter.included.joinToString(",")
+                        excluded = filter.excluded.joinToString(",")
+                    }
+
+                    else -> {}
+                }
+            }
+        }
+
+        return GET("https://readcomic.net/advanced-search?key=${query.replace(" ","+")}&wg=$included&wog=$excluded&status=$statusFilterText&page=$page")
+    }
+
+    override fun searchMangaSelector() = popularMangaSelector()
+
+    override fun searchMangaFromElement(element: Element): SManga {
+        return popularMangaFromElement(element)
+    }
+
+    override fun searchMangaNextPageSelector() = popularMangaNextPageSelector()
+
+    override fun mangaDetailsParse(document: Document): SManga {
+        val infoElement = document.selectFirst(".manga-details tbody")
+        val manga = SManga.create()
+        manga.title = infoElement!!.child(0).child(1).text()
+        manga.author = infoElement.child(4).child(1).text()
+        manga.genre = infoElement.child(5).child(1).text()
+        manga.description = document.select(".pdesc").first()!!.text()
+        manga.status = infoElement.child(3).child(1).text().orEmpty().let { parseStatus(it) }
```

Comment:
Use the `:contains()` jsoup selector in case these fields are reordered in an update.
For reference: https://jsoup.org/apidocs/org/jsoup/select/Selector.html